src/order/conditionally_complete_lattice.lean

```diff
@@ -251,6 +251,24 @@ lemma is_glb_cinfi_set {f : β → α} {s : set β} (H : bdd_below (f '' s)) (Hn
   is_glb (f '' s) (⨅ i : s, f i) :=
 @is_lub_csupr_set αᵒᵈ _ _ _ _ H Hne
 
+lemma csupr_le_iff [nonempty ι] {f : ι → α} {a : α} (hf : bdd_above (range f)) :
+  supr f ≤ a ↔ ∀ i, f i ≤ a :=
+(is_lub_le_iff $ is_lub_csupr hf).trans forall_range_iff
+
+lemma le_cinfi_iff [nonempty ι] {f : ι → α} {a : α} (hf : bdd_below (range f)) :
+  a ≤ infi f ↔ ∀ i, a ≤ f i :=
+(le_is_glb_iff $ is_glb_cinfi hf).trans forall_range_iff
+
+lemma bcsupr_le_iff {ι : Type*} {s : set ι} {f : ι → α} {a : α} (hs : s.nonempty)
```

Comment:
Is this naming convention new? If so, I think it needs to be mentioned in the module docstring

Reply:
Don't really like this name, but okay.